.github/workflows/build-sanitizer-image.yml

```
@@ -0,0 +1,37 @@
+name: Build sanitizer image
+
+on:
+  workflow_dispatch:
+  push:
+  workflow_run:
+    workflows: ["Build image"]
+    types:
+      - completed
+
+jobs:
+    docker:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)